package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bufferapp/web-components",
-  "version": "0.0.16",
+  "version": "0.1.0",
```

Comment:
Looking forward to chatting about version updates :D - would be great to be on the same page here :100: